magda-web-client/src/UI/DataQualityTooltip.js

```diff
@@ -25,7 +25,9 @@ export default function DataQualityTooltip(props) {
             {() => (
                 <React.Fragment>
                     Calculated using the{" "}
-                    <Link to="/page/dataset-quality">Open data scale</Link>
```

Comment:
Should probably avoid breaking the existing URL if we can - can you put a redirect in from /dataset-quality to /linked-data-rating?

Reply:
👍 Added redirection in `OtherPage.js` please have a look when got a chance~